executor/show.go

```diff
@@ -339,6 +342,37 @@ func (e *ShowExec) fetchShowBind() error {
 	return nil
 }
 
+func (e *ShowExec) fetchShowBindingCacheStatus(ctx context.Context) error {
+	exec := e.ctx.(sqlexec.RestrictedSQLExecutor)
+
+	rows, _, err := exec.ExecRestrictedSQL(ctx, nil, fmt.Sprintf("SELECT count(*) FROM mysql.bind_info where status = '%s' or status = '%s' or status = '%s';", bindinfo.Enabled, bindinfo.Using, bindinfo.Disabled))
```

Comment:
So is it possible that bindings on a single SQL are counted multiple times?
Seems the the logic below only counts the enabled bindings.

Reply:
No, a single SQL can only contain one binding at most. Change it to only count the enabled binding.